apps/website/src/components/homepage/__snapshots__/HomeHeroContent.test.tsx.snap

```diff
@@ -376,63 +376,63 @@ exports[`HomeHeroContent > renders as expected 1`] = `
                     <img
                       alt="AI Security Institute"
                       class="h-6"
-                      src="/images/third-party-logos/aisi.png"
+                      src="/images/third-party-logos/aisi.webp"
```

Comment:
svgs would be ideal here, but they mostly weren't readily available, and just compressing to 2x the rendered size achieves the intended effect